src/components/screens/wallet/transactions/transactions.css

```diff
@@ -1,5 +1,17 @@
 @import '../../../../app/mixins.css';
 
+.wrapper {
+  & > header {
+    justify-content: flex-end !important;
+    padding: 0 !important;
+    height: 20px !important;
```

Comment:
I would prefer to avoid using `!important` if possible.

Reply:
The rule that overrides this has so many levels, it's unreasonably hard to override it in this file.